spark-statsd-reporter/src/test/java/org/apache/spark/metrics/sink/statsd/MetricFormatterTest.java

```diff
@@ -0,0 +1,73 @@
+package org.apache.spark.metrics.sink.statsd;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.apache.spark.metrics.sink.statsd.MetricType.GAUGE;
+
+
+
+public class MetricFormatterTest {
+
+    @Test
+    public void simpleDriverMetricString() {
```

Comment:
This all looks great but I don't think we're actually running unit tests for spark-build?

Reply:
Looks like they should be running, there is a target for it in the Makefile

https://github.com/mesosphere/spark-build/blob/master/Makefile#L47